client/components/product-list-item.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import numeral from 'numeral';
+
+export default class ProductListItem extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = null;
+  }
+
+  render() {
+
+    return this.props.products.map(eachProduct => {

```

Comment:
So the best way to handle this would actually be to move your map function up to your product list component. And then you would pass the data for each product in the list into an instance of the product list item. That way each list item is only handling one product. That will avoid issues in the future as you keep building out :)